app/build.gradle

```diff
@@ -88,13 +88,6 @@ dependencies {
     androidTestImplementation 'org.hamcrest:hamcrest-integration:1.3'
 
     androidTestImplementation 'com.github.tomakehurst:wiremock-standalone:2.26.3'
-
-    androidTestImplementation "io.jsonwebtoken:jjwt-api:${rootProject.jsonWebTokenVersion}"
```

Comment:
Can we remove `jsonWebTokenVersion` from the root build.gradle now?

Reply:
Not yet, it us still used with unit tests